sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeServiceClient.cs

```diff
@@ -761,6 +761,195 @@ public virtual DataLakeFileSystemClient GetFileSystemClient(string fileSystemNam
         }
         #endregion Delete File System
 
+        #region Get Service Properties
```

Comment:
I added `DataLakeServiceClient.GetProperties()` and `.SetProperties()` so customers can programmatically enable and disable HNS soft delete using the Data Lake SDK.

Note that `DataLakeServiceProperties` does not contain `StaticWebsite`.  This is by design, I found during testing that HNS accounts do not support `StaticWebsite`.